assets/i18n/en.json

```
@@ -773,8 +773,12 @@
     "I18N_TOPNAV_ADMIN_PAGE": "Admin Page",
     "I18N_TOPNAV_BLOG": "Blog",
     "I18N_TOPNAV_BLOG_DASHBOARD": "Blog Dashboard",
-    "I18N_TOPNAV_CLASSROOM": "Classroom",
-    "I18N_TOPNAV_CLASSROOM_BASIC_MATHS": "Basic Mathematics",
+    "I18N_TOPNAV_CLASSROOM": "Learn",
```

```suggestion
    "I18N_TOPNAV_LEARN": "Learn",
```
As it is does not only contain classroom section. Ditto to all other places where we are talking about the whole dropdown and not just about the classroom section.

---

Done